src/main/scala/evaluation/Evaluator.scala

```diff
@@ -0,0 +1,11 @@
+package evaluation
+
+import org.apache.spark.rdd.RDD
+import workflow.PipelineDataset
+
+trait Evaluator[P,L,E] {
```

Comment:
I actually posted this comment on friday, but I forgot to press 'submit review' so I guess it didn't post until now...